webapp/channels/src/components/admin_console/license_settings/enterprise_edition/enterprise_edition_right_panel.tsx

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import * as React from 'react';
+import React, {memo} from 'react';
```

Comment:
Yeah we dont have to.

Reply:
I'm pretty sure you need to import `React` itself for all the JSX syntax to work since it gets converted to `React.createElement` by the tooling. My information on that could be very dated though

Reply:
I'm the one probably dated here so just ignore me 🤐